src/cpp/database/database.cpp

```diff
@@ -426,15 +531,18 @@ void Database::insert(
         }
         case DataKind::NETWORK_LATENCY:
         {
-            /* Check that the entity is a known locator */
-            auto locator = locators_[entity_id];
-            if (locator)
-            {
-                const NetworkLatencySample& network_latency = dynamic_cast<const NetworkLatencySample&>(sample);
-                locator->data.network_latency_per_locator[network_latency.remote_locator].push_back(network_latency);
-                break;
-            }
-            throw BadParameter(std::to_string(entity_id.value()) + " does not refer to a known locator");
+            const NetworkLatencySample& network_latency = dynamic_cast<const NetworkLatencySample&>(sample);
+
+            // Create locator if it does not exists
```

Comment:
```suggestion
            // Create locator if it does not exist
```

There are several other comments that uses `does not exists`. Please, correct this typo in all of them.